app/models/pacbio/well.rb

```diff
@@ -60,12 +54,6 @@ class Well < ApplicationRecord # rubocop:disable Metrics/ClassLength
     delegate :run, to: :plate, allow_nil: true
 
     def pool_ids=(ids)
```

Comment:
I think pool_ids an library_ids should be replaced by sending the used_aliquots directly. As now pools and libraries are no longer directly associated this doesn't really make sense. This would require changes to the UI as well which is why I haven't changed it yet. Probably worth doing when we do the well volume tracking work is done as that will require aliquots in the UI.

Reply:
https://github.com/sanger/traction-ui/issues/1518